commitlint.config.ts

```diff
@@ -27,7 +30,7 @@ module.exports = {
         'body-soft-max-line-length': [RuleStatus.Error, 'always'],
         'footer-leading-blank': [RuleStatus.Warning, 'always'],
         'footer-max-line-length': [RuleStatus.Error, 'always', 150],
-        'header-max-length': [RuleStatus.Error, 'always', 50],
+        'header-max-length-with-suggestions': [RuleStatus.Error, 'always'],
```

Comment:
why remove 50 as a parameter? I mean, it's good that you created var headerMaxLineLength, but it would be better if we can pass it as a parameter to the rule config

Reply:
Done